proto/sifnode/tokenregistry/v1/types.proto

```diff
@@ -58,6 +58,7 @@ message RegistryEntry {
   // the packet level. i.e rowan -> microrowan i.e microrowan -> microrowan
   string ibc_counterparty_denom = 17;
   string ibc_counterparty_chain_id = 18;
+  repeated sifnode.oracle.v1.NetworkDescriptor double_pegged_networks = 19;
```

Comment:
Should this be a mapping? So that we can lookup by network id if its been double pegged or not?

Reply:
the map also works, the array is simpler than a map. we put the network descriptor in the array after it is double pegged.